src/plugins/data/common/index.ts

```diff
@@ -22,5 +22,5 @@ export * from './field_formats';
 export * from './kbn_field_types';
 export * from './index_patterns';
 export * from './es_query';
-
+export * from './utils';
```

Comment:
This means that `shortenDottedString` is still going to be leaked into the public contract since `data/public/index.ts` does an `export * from '../common';`

I'm going to open a follow-up PR to address this more widely, so we can ensure nothing is being exported out of common that doesn't belong in our public API